### PR TITLE
react-week1

### PR DESCRIPTION
**Pull request by suvarna-ratna**
waiting for feedback

### CODE REVIEW

File: react/week1/todo-app/src/todo.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      id: 2,
+      description: "Brush teeth",
+      deadline: "Thu Sep 14 2017",
+    },
+    {
+      id: 3,
+      description: "Eat breakfast",
+      deadline: " Fri Sep 15 2017",
+    },
+  ];
```
Comment (islanderupnorth) [inline react/week1/todo-app/src/todo.js:20]: A minor comment but for static constants/variables then it's a good idea to keep them outside the functions that renders as it won't change

File: react/week1/todo-app/src/todo.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  console.log(todoDetails);
+  return (
+    <div>
+      <h1>Todo List</h1>
+      <ul>
+        {todoDetails.map((todo, i) => (
+          <li key={i}>
+            {todo.description}
+            <span> | </span> {todo.deadline}
+          </li>
```
Comment (islanderupnorth) [inline react/week1/todo-app/src/todo.js:30]: If you can, try break the parent ToDoList and ToDoItems into separate components

**Review by islanderupnorth — COMMENTED** [review]
It looks good overall & the basics are in place with the right props. I'd recommend trying to breaking the list & item into separate components and pass the props through but good work!

**Comment by suvarna-ratna** [comment]
thank you

**Pull request merged**